tests/components/myuplink/conftest.py

```diff
@@ -107,30 +108,30 @@ def system_fixture(load_systems_file: str) -> list[System]:
 # Fixture group for device points API endpoint.
 
 
-@pytest.fixture(scope="session")
+@pytest.fixture
```

Comment:
Why did we change the fixture scopes and json loading? This was optimized to avoiding loading from disk more than once.

Reply:
Oh I removed it because the json loading was moved to another fixture but i forgot to place it back